draft-ietf-quic-http.md

```diff
@@ -1339,7 +1345,7 @@ error of type H3_FRAME_UNEXPECTED.
 CANCEL_PUSH Frame {
   Type (i) = 0x3,
   Length (i),
-  Push ID (..),
+  Push ID (..8),
```

Comment:
I was a bit torn on this one, since the unrestricted form matches the generic frame spec more clearly.  OTOH, we have other frame specifications that just put their own specific stuff here, so I figured I would see what we thought.

Reply:
```suggestion
  Push ID (i),
```

This is just a varint.  I don't know why this was `(..)` rather than `(i)`.  Probably my fault.

Reply:
Good point; this is better.